includes/plugin.php

```diff
@@ -706,6 +712,7 @@ private function __construct() {
 		$this->register_autoloader();
 
 		$this->logger = Log_Manager::instance();
+		$this->dev_tools = Dev_Tools::instance();
```

Comment:
Is the `dev_tools` a Plugin property or a common module?

Reply:
Remove it from here.

Reply:
Handled.